cmd/gaia/cmd/gaiacli/main.go

```diff
@@ -66,6 +67,7 @@ func main() {
 		tendermintCmd,
 		ibcCmd,
 		lcd.ServeCommand(cdc),
+    lcd.ServeSwaggerCommand(cdc),
```

Comment:
Please add to Basecoin's [basecli](https://github.com/cosmos/cosmos-sdk/blob/develop/examples/basecoin/cmd/basecli/main.go) file as well